hypervisor/arch/x86/guest/ucode.c

```diff
@@ -50,8 +50,8 @@ uint64_t get_microcode_version(void)
  * According to SDM vol 3 Table 9-7. If data_size field of uCode
  * header is zero, the ucode length is 2000
  */
-#define	GET_DATA_SIZE(hdptr)	\
-	((hdptr)->data_size ? ((hdptr)->data_size) : 2000)
+#define	GET_DATA_SIZE(hdptr)	((hdptr)->data_size ?\
+			((hdptr)->data_size) : 2000)
```

Comment:
run checkpatch.pl.  
BTW, it seems you bring spaces here. This is is not allowed. Use TAB. Please check the coding style ..